### PR TITLE
Format README

### PR DESCRIPTION
**Pull request by ivyostosh**
Update the format and language of README to improve readability

*Issue #, if available:* https://github.com/aws/eks-anywhere-packages/issues/163

*Description of changes:* `README.md`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,18 +14,18 @@ The EKS Anywhere package controller and the EKS Anywhere Curated Packages (refer
 
 ---
```
Comment (TerryHowe) [inline README.md:16]: I'm not a big fan of "The" here. It works for "The Batman", but not "The Terry Howe"

Comment (ivyostosh) [reply README.md:16]: I agree. This is the original approved verbiage though, so I don't want to mess up with it. I can update the rest of the README file to not use "the" convention. Does that work?

Comment (TerryHowe) [reply README.md:16]: Leave the disclaimer as it is, but we don't need the "the".

Comment (ivyostosh) [reply README.md]: Updated all the references except the disclaim itself.

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Pull request merged**